src/bout++.cxx

```diff
@@ -218,6 +218,7 @@ namespace experimental {
 void setupSignalHandler(SignalHandler signal_handler) {
 #if BOUT_USE_SIGNAL
   std::signal(SIGSEGV, signal_handler);
+  std::signal(SIGBUS, signal_handler);
```

Comment:
It is not a workaround.
While SIGBUS is far not that common as SIGSEGV, I think it makes sense to catch this as well and generate a backtrace. Or do you think there are cases where we should not handle a BUS error, and the code can recover from that?
I guess if some other code is just using some parts of BOUT++, it might make sense to not handle, it, but in that case I would argue it should be configured with `-DBOUT_ENABLE_SIGNAL=OFF`

Anyway, it is not often useful, so happy to revert.

Reply:
Yes, I've thought about it some more and you're right, this should be rare but useful to catch.